### PR TITLE
Allow .tgz extensions to pass validation

### PR DESCRIPTION
**Pull request by Lukazar**
.tgz is a valid extension.

### CODE REVIEW

**Comment by DavertMik** [comment]
Thanks. Good addition.

This line was not really nice but with this patch it will look really ugly :cry: 

```php
 if (strlen($this->dumpFile) <= 7 || (substr($this->dumpFile, -7) !== '.tar.gz' && substr($this->dumpFile, -4) !== '.tgz')) {
```
Could you think how to simplify this condition check so it would look nicer?

File: src/Codeception/Module/MongoDb.php [hunk 1]
```diff
@@ -158,7 +158,7 @@ private function validateDump()
                         "Tar gunzip archives are not supported for Windows systems"
                     );
                 }
-                if (strlen($this->dumpFile) <= 7 || (substr($this->dumpFile, -7) !== '.tar.gz' && substr($this->dumpFile, -4) !== '.tgz')) {
+                if (!preg_match('/^.+(\.tar\.gz|\.tgz)$/', $this->dumpFile)) {
```
Comment (DavertMik) [inline src/Codeception/Module/MongoDb.php]: `^.` what about space characters?
I think regex can be simplified to just `/(\.tar\.gz|\.tgz)$/`

Comment (Lukazar) [reply src/Codeception/Module/MongoDb.php]: You bet.

Comment (DavertMik) [reply src/Codeception/Module/MongoDb.php]: Yes, sorry, space character is included

Comment (Lukazar) [reply src/Codeception/Module/MongoDb.php]: What you have should be fine so long you do not care about files that are named just ".tgz".  To be honest the check isn't really needed.  The system will puke if you try to untar something thats not a valid archive... I think... I didn't test it explicitly.

Comment (DavertMik) [reply src/Codeception/Module/MongoDb.php]: Yeah, got it, thanks. Once build finishes I'm merging

**Pull request merged**